PluginCore/ScintillaNet/ScintillaControl.cs

```diff
@@ -3190,10 +3190,9 @@ public void GotoPos(int pos)
         /// </summary>
         unsafe public string GetCurLine(int length)
         {
-            int sz = SPerform(2027, length, 0);
-            byte[] buffer = new byte[sz + 1];
+            byte[] buffer = new byte[length + 1];
             fixed (byte* b = buffer) SPerform(2027, length + 1, (uint)b);
-            return Encoding.GetEncoding(this.CodePage).GetString(buffer, 0, sz - 1);
+            return Encoding.GetEncoding(this.CodePage).GetString(buffer, 0, length);
```

Comment:
Aha, you're right. 
Also I think it would be better to cap `length` to `sz - 1` at the start instead, so `SPerform` also doesn't waste memory&time by writing extra null characters that will be ignored.